components/BottomAppBar/src/private/MDCBottomAppBarLayer.m

```diff
@@ -102,12 +101,12 @@ - (UIBezierPath *)drawWithPlainPath:(UIBezierPath *)bottomBarPath
                               width:(CGFloat)width
                              height:(CGFloat)height
                           arcCenter:(CGPoint)arcCenter
-                   hypotenuseLength:(CGFloat)hypotenuseLength {
-  CGFloat halfOfHypotenuseLength = hypotenuseLength / 2;
+                          arcRadius:(CGFloat)arcRadius {
   [bottomBarPath moveToPoint:CGPointMake(0, yOffset)];
-  [bottomBarPath addLineToPoint:CGPointMake(arcCenter.x - halfOfHypotenuseLength, yOffset)];
+  [bottomBarPath addLineToPoint:CGPointMake(arcCenter.x - arcRadius, yOffset)];
   [bottomBarPath addLineToPoint:CGPointMake(arcCenter.x, yOffset)];
-  [bottomBarPath addLineToPoint:CGPointMake(arcCenter.x + halfOfHypotenuseLength, yOffset)];
+  [bottomBarPath addLineToPoint:CGPointMake(arcCenter.x + arcRadius, yOffset)];
+  [bottomBarPath addLineToPoint:CGPointMake(arcCenter.x + arcRadius, yOffset)];
```

Comment:
If these are going to be the exact same lines, please add a comment to remark that they are intentional (and explain why) and not an accidental line paste.  Maintainers are likely to see this as a no-op and attempt to remove it.